include/libopencm3/stm32/common/dma_common_l1f013.h

```diff
@@ -138,6 +138,11 @@ specific memorymap.h header before including this header file.*/
 #define DMA2_CMAR4			DMA2_CMAR(DMA_CHANNEL4)
 #define DMA2_CMAR5			DMA2_CMAR(DMA_CHANNEL5)
 
+/* DMA channel selection register (DMAx_CSELR) */
+#define DMA_CSELR(dma_base)		MMIO32((dma_base) + 0xA8)
+#define DMA1_CSELR			DMA_CSELR(DMA1)
+#define DMA2_CSELR			DMA_CSELR(DMA2)
+
```

Comment:
These should be in the f0 file, not the l1f013 file, this register doesn't exist on l1, f1, nor f3

Reply:
Thanks for the correction. I pushed a fixup commit (to be merged with the previous commit), but I can also force-push the branch if you prefer.

Reply:
I'm happy to squish when I merge, no problem